Tropos/Models/DailyForecast.swift

```diff
@@ -0,0 +1,15 @@
+import Foundation
+
+@objc class DailyForecast: NSObject {
+    let date: NSDate
+    let conditionsDescription: String
+    let highTemperature: Temperature
+    let lowTemperature: Temperature
+    
+    init(JSON: Dictionary<String, AnyObject>) {
```

Comment:
Hey @jakecraige and @gfontenot - is that possible to discuss this topic more in real time so we can decide about preferred solution faster? Some Slack, Skype, Hangouts, Messages or something?